src/cpp/desktop/DesktopGwtCallbackMac.mm

```diff
@@ -460,7 +461,26 @@ bool showOfficeDoc(NSString* path, NSString* appName, NSString* formatString)
    }
 }
 
+void GwtCallback::changeTitleBarColor(int red, int green, int blue) {
+    WId winId = pOwner_->asWidget()->effectiveWinId();
+    if (winId == 0) return;
+    NSView* view = (NSView*)winId;
+    NSWindow* window = [view window];
+    if (pow(red, 2.) + pow(green, 2.) + pow(blue, 2.) < 48387) {
```

Comment:
Where does the magic number 48387 come from? (Just as a curiosity; not necessarily requiring any change in the PR)

Reply:
127^2 *3  The sum of average color!?

Reply:
For clarity, it would be worth encoding this as a constant, e.g. `const int kAverageColor = pow(127.0, 2.0) * 3`.

Reply:
Definitely. It may be easier for you to commit.